packages/flutter/lib/src/gestures/pointer_router.dart

```diff
@@ -14,8 +14,8 @@ typedef PointerRoute = void Function(PointerEvent event);
 
 /// A routing table for [PointerEvent] events.
 class PointerRouter {
-  final Map<int, LinkedHashSet<_RouteEntry>> _routeMap = <int, LinkedHashSet<_RouteEntry>>{};
-  final LinkedHashSet<_RouteEntry> _globalRoutes = LinkedHashSet<_RouteEntry>();
+  final Map<int, LinkedHashMap<PointerRoute, Matrix4>> _routeMap = <int, LinkedHashMap<PointerRoute, Matrix4>>{};
```

Comment:
Instead of LinkedHashMap, can you just use Map and when instantiating use the map literal { }? LinkedHashMap is the default implementation that you get there and it reads a lot nicer.

Reply:
(This applies everywhere in this code)

Reply:
Fixed.